client/src/components/Footnote.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { theme } from "../constants/theme";
+
+import styled from "styled-components";
+
+const { colors } = theme;
+
+const StyledFooter = styled.div`
+  position: absolute;
```

Comment:
Not sure if using position absolute is appropriate for this actually. Using that along with top/bottom` and `left/right` values might not make the footnote responsive, since they are fixed values right now. You could probably see what I mean by changing the screen size and see how the links move in response. I think we could just give the footer-links to have display inline-block so that they are all on same line, and then just mess with the margin/padding to get the right amount of spacing. Let's also remove the width and height values for each of the different links, since those values with be determined by the content. Let me know if you have any questions with that.